src/Facades/ChildProcess.php

```diff
@@ -6,11 +6,13 @@
 
 /**
  * @method static \Native\Laravel\ChildProcess[] all()
- * @method static \Native\Laravel\ChildProcess get(string $alias)
+ * @method static \Native\Laravel\ChildProcess get(string $alias = null)
  * @method static \Native\Laravel\ChildProcess message(string $message, string $alias = null)
- * @method static \Native\Laravel\ChildProcess restart(string $alias)
- * @method static \Native\Laravel\ChildProcess start(string $alias, array $cmd, string $cwd = null, array $env = null, bool $persistent = false)
- * @method static void stop(string $alias)
+ * @method static \Native\Laravel\ChildProcess restart(string $alias = null)
+ * @method static \Native\Laravel\ChildProcess start(string|array $cmd, string $alias, string $cwd = null, array $env = null, bool $persistent = false)
+ * @method static \Native\Laravel\ChildProcess php(string|array $cmd, string $alias, array $env = null, bool $persistent = false)
+ * @method static \Native\Laravel\ChildProcess artisan(string|array $cmd, string $alias, array $env = null, bool $persistent = false)
+ * @method static \Native\Laravel\ChildProcess void stop(string $alias = null)
```

Comment:
```suggestion
 * @method static void stop(string $alias = null)
```

No need for the extra class reference here - the return type is `void`

Reply:
yep. slight oversight on my part